Formula/guile.rb

```diff
@@ -35,26 +41,39 @@ class Guile < Formula
   depends_on "pkg-config" # guile-config is a wrapper around pkg-config.
   depends_on "readline"
 
+  ### Remove after configure.ac patch is not necessary anymore.
+  uses_from_macos "flex" => :build
+  ###
+
   uses_from_macos "gperf"
   uses_from_macos "libffi", since: :catalina
   uses_from_macos "libxcrypt"
 
+  patch do
```

Comment:
Patch should only be applied to stable

Reply:
Done. I followed another formula, let me know if that's the right way. Thanks!